mindjunkies/courses/forms.py

```
@@ -2,59 +2,56 @@
 from django import forms
 from django.utils.text import slugify
 
-from .models import Course, CourseObjective, CourseRequirement
+from .models import Course, CourseToken
 
+from django.forms import inlineformset_factory
+from .models import Course, CourseInfo
```

Ruff (0.8.2)

Redefinition of unused `Course` from line 5

Remove definition: `Course`

(F811)